Aplib.Tests/Belief/BeliefTests.cs

```diff
@@ -102,4 +107,61 @@ public void UpdateBelief_ReferenceIsAssignedTo_DoesNotUpdateObservation()
         Assert.NotEqual(def, belief);
         Assert.NotEqual(def, belief.Observation);
```

Comment:
Implicit conversion is tested in another test.
```suggestion
        Assert.NotEqual(def, belief);
```